File: Ruby/tests/problem061_test.rb
require 'minitest/autorun'
require_relative '../problem061'

class Problem063Test < Minitest::Test
  def test_alt_pow_returns_correct_result
    assert 2.alt_pow(10) == 1024
    assert 3.alt_pow(4) == 81
  end
end

File: JavaScript/tests/problem061_test.js
const chai = require('chai')
const assert = chai.assert
const problem061 = require('../problem061');
const canFormWord = problem061.canFormWord;

// Implement kind of a factory function
function getTestMatrix() {
    return [['F', 'A', 'C', 'I'],
            ['O', 'B', 'Q', 'P'],
            ['A', 'N', 'O', 'B'],
            ['M', 'A', 'S', 'S']]
}

describe('Problem 61', function(){
  it("calculates the correct result", function() {
    assert.isTrue(problem061.altPow(2, 10) === 1024)
    assert.isTrue(problem061.altPow(3, 4) === 81)
  });
});
